client/src/App.jsx

```diff
@@ -25,9 +27,10 @@ function App() {
 
         <Route path="dashboard/*" element={<Dashboard />}></Route>
 
-        <Route path="/login/" element={<Login />}></Route>
-        <Route path="/register/" element={<Register />}></Route>
-      </Routes>
+          <Route path='/login/' element={<Login />}></Route>
+          <Route path='/register/' element={<Register />}></Route>
+          <Route path='/createQuiz/' element={<CreateQuiz />}></Route>
```

Comment:
Suggestion: Update the Create Quiz link in the Nav bar to match this new route-- that way users can click to access the Create Quiz page